language/language_samples.rb

```diff
@@ -21,9 +21,11 @@ def sentiment_from_text project_id:, text_content:
 
   language  = Google::Cloud::Language.new project: project_id
   document  = language.document text_content
-  sentiment = document.sentiment
-
-  puts "#{sentiment.score} (#{sentiment.magnitude})"
+  puts "Overall document sentiment: (#{document.sentiment.score})"
+  puts "Sentence level sentiment:"
+  document.sentiment.sentences.each do |sentence|
+    puts "#{sentence.text}: (#{sentence.sentiment.score})"
```

Comment:
Tests should be added to verify multiple sentences were identified by the sample.